Can't be used with System.Text.Json source generation

While working on a project using ActivityStreams I thought I'd try using the new System.Text.Json source generation.

Unfortunately there's an issue because the converter types are internal.

```csharp
[JsonSerializable(typeof(Person))]
public partial class ActivityStreamJsonSerializerContext : JsonSerializerContext {}
```

Produces the following compiler errors.

> error CS0122: 'LinkConverter' is inaccessible due to its protection level
> error CS0122: 'EndpointsOrLinkConverter' is inaccessible due to its protection level
> error CS0122: 'ObjectOrLinkConverter' is inaccessible due to its protection level
> error CS0122: 'ImageOrLinkConverter' is inaccessible due to its protection level
> error CS0122: 'CollectionPageOrLinkConverter' is inaccessible due to its protection level
> error CS0122: 'TermDefinitionConverter' is inaccessible due to its protection level
> error CS0122: 'XMLTimeSpanConverter' is inaccessible due to its protection level

Could the converters please be changed to public?

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/CollectionOrLinkConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/CollectionOrLinkConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class CollectionOrLinkConverter : JsonConverter<ICollectionOrLink?>
+public class CollectionOrLinkConverter : JsonConverter<ICollectionOrLink?>
 {
     public override ICollectionOrLink? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/CollectionPageOrLinkConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/CollectionPageOrLinkConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class CollectionPageOrLinkConverter : JsonConverter<ICollectionPageOrLink?>
+public class CollectionPageOrLinkConverter : JsonConverter<ICollectionPageOrLink?>
 {
     public override ICollectionPageOrLink? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/DateTimeBooleansObjectOrLinkConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/DateTimeBooleansObjectOrLinkConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class DateTimeBooleanObjectOrLinkConverter : JsonConverter<DateTimeBooleanObjectOrLink?>
+public class DateTimeBooleanObjectOrLinkConverter : JsonConverter<DateTimeBooleanObjectOrLink?>
 {
     public override DateTimeBooleanObjectOrLink? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/EndpointsOrLinkConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/EndpointsOrLinkConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class EndpointsOrLinkConverter : JsonConverter<IEndpointsOrLink?>
+public class EndpointsOrLinkConverter : JsonConverter<IEndpointsOrLink?>
 {
     public override IEndpointsOrLink? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/ImageOrLinkConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/ImageOrLinkConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class ImageOrLinkConverter : JsonConverter<IImageOrLink?>
+public class ImageOrLinkConverter : JsonConverter<IImageOrLink?>
 {
     public override IImageOrLink? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/LinkConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/LinkConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class LinkConverter : JsonConverter<ILink?>
+public class LinkConverter : JsonConverter<ILink?>
 {
     public override ILink? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/ObjectConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/ObjectConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class ObjectConverter : JsonConverter<IObject?>
+public class ObjectConverter : JsonConverter<IObject?>
 {
     public override IObject? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/ObjectOrLinkConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/ObjectOrLinkConverter.cs
@@ -4,7 +4,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class ObjectOrLinkConverter : JsonConverter<IObjectOrLink?>
+public class ObjectOrLinkConverter : JsonConverter<IObjectOrLink?>
 {
     public override IObjectOrLink? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/TermDefinitionConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/TermDefinitionConverter.cs
@@ -5,7 +5,7 @@ using static System.Text.Json.JsonSerializer;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class TermDefinitionConverter : JsonConverter<ITermDefinition?>
+public class TermDefinitionConverter : JsonConverter<ITermDefinition?>
 {
     public override ITermDefinition? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/JsonConverters/XMLTimeSpanConverter.cs
+++ b/src/KristofferStrube.ActivityStreams/JsonConverters/XMLTimeSpanConverter.cs
@@ -4,7 +4,7 @@ using System.Xml;
 
 namespace KristofferStrube.ActivityStreams.JsonConverters;
 
-internal class XMLTimeSpanConverter : JsonConverter<TimeSpan?>
+public class XMLTimeSpanConverter : JsonConverter<TimeSpan?>
 {
     public override TimeSpan? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {

--- a/src/KristofferStrube.ActivityStreams/ObjectTypes.cs
+++ b/src/KristofferStrube.ActivityStreams/ObjectTypes.cs
@@ -1,8 +1,8 @@
 ﻿namespace KristofferStrube.ActivityStreams;
 
-internal static class ObjectTypes
+public static class ObjectTypes
 {
-    internal static readonly Dictionary<string, Type> Types = new()
+    public static readonly Dictionary<string, Type> Types = new()
     {
         { "Object", typeof(Object) },
         { "Collection", typeof(Collection) },

